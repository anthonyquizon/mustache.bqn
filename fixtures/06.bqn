data⇐{
  a⇐⟨{c⇐1},{c⇐2},{c⇐3}⟩
  b⇐⟨{c⇐"a"},{c⇐"b"},{c⇐"c"}⟩
}
