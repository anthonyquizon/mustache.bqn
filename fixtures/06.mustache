<x>
    {{#a}}
        <y>{{c}}</y>
    {{/a}}

    {{#b}}
        <z>{{c}}</z>
    {{/b}}
</x>
