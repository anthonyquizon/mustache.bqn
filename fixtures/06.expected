<x>
    <y>1</y>
    <y>2</y>
    <y>3</y>

    <z>a</z>
    <z>b</z>
    <z>c</z>
</x>
